Assets/Build.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
```

Comment:
What's this folder meta? `Assets/Build.meta`

Reply:
No idea. I do a `git clean -df` regularly, but Unity still manages to generate weird files. I'll remove 😄